langgraph/pregel/__init__.py

```diff
@@ -621,8 +632,19 @@ async def aupdate_state(
             # apply to checkpoint and save
             _apply_writes(checkpoint, channels, task.writes)
             step = saved.metadata.get("step", -2) + 1 if saved else -1
+
+            # merge configurable fields with previous checkpoint config
+            checkpoint_config = config
+            if saved:
+                checkpoint_config = {
```

Comment:
We're constructing a new "RunnableConfig" that doesn't have all of the keys defined in `RunnableConfig`. If a developer is implementing the `put()`/`aput()` API for a custom checkpointer, excluding the keys might be unexpected. Is this what we want?

Reply:
All keys are optional